### PR TITLE
got googleAuth working

### PR DESCRIPTION
**Pull request by melikaxo**

### CODE REVIEW

**Review by sumitz1207 — APPROVED** [review]

**Review by smcgovern-proj — CHANGES_REQUESTED** [review]
Hey ty for getting google Auth working! That's super cool. 

There are a couple changes I'd like to request before we merge this into development, if you think that's possible.

- Right now the package-lock is added to the .gitignore, which is probably the source of some merge conflicts. I think the best way to resolve this would be to delete the package-lock, pull down any changes on the development branch, and then merge development with this backend branch?
- I am not sure what the bundle 2.js does bc it won't load on github -- assuming you had to do it to get around a merge conflict or something like that. The good news is, I think we can just delete it (and even the bundle.js if necessary) because webpack should generate a new bundle each time.
- There's just one line in the App.jsx that is different, which is the outer div's className. I think we can solve that by either pulling from development or just copy-pasting from what's there right now.
- Finally, would love to see one last pull/merge with development to make sure this branch is up to date on all the latest PR's before we merge it in.

If any of this sounds unreasonable or you think it's unnecessary can definitely let me know! This is just based on my opinion. Also totally happy to pair program these changes or talk it out with you if you want.

**Review by smcgovern-proj — APPROVED** [review]

**Pull request merged**
